irods_store.py

```diff
@@ -428,16 +443,17 @@ def add(self, image_id, image_file, image_size):
                              'path': self.path,
                              'user': self.user,
                              'password': self.password,
-                             'data_name': image_id})
+                             'data_name': image_id},
+                             None)
         return (loc.get_uri(), bytes_written, checksum_hex, {})
 
     def _option_get(self, param):
         result = getattr(CONF, param)
         if not result:
-            reason = _("Could not find %(param)s in configuration options.") \
+            reason = "Could not find %(param)s in configuration options." \
```

Comment:
Yep, I changed this unnecessarily as part of troubleshooting (before we were importing glance_store.i18n) and never changed it back. Just fixed and pushed again.